### DISCUSSION TITLE
process for reporting a bug

### DISCUSSION THREAD

**Discussion by CorinaMurg** [open-sauced/app#4184] [Q&A] [answered]
My apologies if I missed this info in the Contributing guides. If we come across a bug, how do we bring it up to the maintainers' attention?

**Comment by nickytonline** [comment, marked as answer]
[Create a new issue](https://github.com/open-sauced/app/issues/new/choose) and it will automatically get labelled with "needs triage".

It will appear in the core team members GitHub notifications and we'll take it from there.

See our contributing guide for more info. https://github.com/open-sauced/app/blob/beta/README.md#-contributing
